airflow/dags/utils/random_profile.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+import awswrangler as wr
+import boto3
+from airflow.models import Variable
+
+from .main import fetch_users
+
+all_profiles = fetch_users(3000)
+male_all_profiles = all_profiles[all_profiles["gender"] == "male"]
+female_all_profiles = all_profiles[all_profiles["gender"] == "female"]
+bucket = 'dev-male-female-ingestion'
+col = {"location.postcode": "string"}    # explicit dtype casting to string
+
+
+# Establish connection with AWS and Extract files to AWS S3
+def aws_session():
```

Comment:
Move this to a dedicated module and call it, you can call it `aws.py`, with that we know that all aws related modules will live there